addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -569,6 +578,18 @@ public void scan(HttpMessage msg, String param, String value) {
         }
     }
 
+    private static boolean isGoodCandidate(HttpMessage msg) {
+        if (ResourceIdentificationUtils.isJavaScript(msg)
+                || ResourceIdentificationUtils.isCss(msg)
+                || ResourceIdentificationUtils.responseContainsControlChars(msg)) {
+
+            return false;
+        }
+        return DirNamesContentsMatcher.matchNixDirectories(msg.getResponseBody().toString()) == null
```

Comment:
It would be better if this was done separately for the OS that's being checked, there's no reason to skip because of Nix occurrences if we are checking Win.

Reply:
I believe this has been addressed.